recipes/fbpic/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set version = "0.25.0" %}
+
+package:
+  name: fbpic
+  version: {{ version }}
+
+source:
+  fn: {{ version }}.tar.gz
+  url: https://github.com/fbpic/fbpic/archive/refs/tags/{{ version }}.tar.gz
+  sha256: a868cae099fac66d20017d405caea17cb3f1aa00cb83489038f4474ca8fd28f3
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+    - setuptools
+  run:
+    - h5py
+    - numba
+    - numpy
+    - python >=3.6
+    - python-dateutil
+    - scipy
+# pyfftw
```

Comment:
Can you remove or add a comment on why this is commented out?

Reply:
```suggestion
# For fast non-MKL FFTs, explore using in the future:
# pyfftw
```